ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/models/Element.java

```diff
@@ -41,4 +57,16 @@ public String getType() {
     public String getComponentType() {
         return componentType;
     }
+
+    public String getXmlTagName() {
+        return xmlTagName;
+    }
+
+    public boolean isAsAttribute() {
+        return asAttribute;
+    }
+
+    public boolean hasWrapper() {
```

Comment:
While I would normally agree what `isWrapper` is the correct way to do it.  `hasWrapper` makes more sense to me.